bitcoin/src/psbt/mod.rs

```diff
@@ -818,6 +818,7 @@ mod display_from_str {
     use core::str::FromStr;
 
     use base64::display::Base64Display;
+    use base64::prelude::{Engine as _, BASE64_STANDARD as base64_std};
```

Comment:
At the risk of bike shedding; I'm not super keen on this alias because:
1. The identifier `base64_std` makes me think of the "std" feature and then think "what has this got to do with std and why is there no feature gate if its std"?
2. It removes the const information that is implied by the all caps and makes me have to look around the code to work out what the `base64_std` identifier represents.

Anyway, its only in a small scope so I don't really mind but wanted to share the thoughts.